### PR TITLE
[Merged by Bors] - chore: release 0.10.9

### PR DESCRIPTION
**Pull request by nacardin**

### CODE REVIEW

**Comment by morenol** [comment]
@nacardin can we wait for https://github.com/infinyon/fluvio/pull/3247 first?

I am testing a fix to cdk deploy that is affecting mqtt-connector (becuase it is using latest version of cdk)

**Comment by nacardin** [comment]
> @nacardin can we wait for #3247 first?
> 
> I am testing a fix to cdk deploy that is affecting mqtt-connector (becuase it is using latest version of cdk)

ok

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,34 @@
...
+### Added
+
+* Simplified control plane api ([#3073](https://github.com/infinyon/fluvio/issues/3073))
+* Add helper method to get utf8 from data ([#3236](https://github.com/infinyon/fluvio/issues/3236))
+* Move x records outputed from smdk test to --verbose #3169 ([#3234](https://github.com/infinyon/fluvio/issues/3234))
+* Add message about record too large on produce cli ([#3242](https://github.com/infinyon/fluvio/issues/3242))
+
+### CI
+
+* `cdk build` uses a default target ([#3222](https://github.com/infinyon/fluvio/issues/3222))
```
Comment (morenol) [inline CHANGELOG.md:14]: update to include last commit: `cdk test` and `cdk deploy` also uses default target

**Review by morenol — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (nacardin) [reply CHANGELOG.md:14]: updated

**Pull request closed without merging**